web/templates/message_view_header.hbs

```diff
@@ -1,9 +1,9 @@
 {{#if stream_settings_link}}
-<a class="message-header-stream-settings-button" href="{{stream_settings_link}}">
+<a class="message-header-stream-settings-button" href="{{stream_settings_link}}" data-stream-text="{{stream_tooltip_text}}" data-sub-count-text="{{sub_count_tooltip_text}}">
```

Comment:
Since we are already passing `stream_tooltip_text` and `sub_count_tooltip_text` into this template, we can use these variables inside a tooltip template, like such:
```handlebars
<template id="stream-details-tooltip-template">
    <div>
        <div>{{stream_tooltip_text}}</div>
        <div class="tooltip-inner-content">{{sub_count_tooltip_text}}</div>
    </div>
</template>
```
and then reference this template from the stream heading element. This would allow us to avoid the code complexity in `click_handlers.js`